octopoes/octopoes/xtdb/client.py

```diff
@@ -155,16 +145,16 @@ def create_node(self) -> None:
         try:
             res = self._session.post("/create-node", json={"node": self._client})
             self._verify_response(res)
-        except HTTPError as e:
+        except HTTPStatusError as e:
```

Comment:
I might have overlooked this one, but I think this should also be `HTTPError` to properly log and raise XTDBException in case any other problem happens in the connection with XTDB.